src/main/java/HM05/FruitMarket.java

```diff
@@ -0,0 +1,62 @@
+package HM05;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+
+public class FruitMarket {
+    List<Fruits> listOfFruits = new ArrayList<>();
+
+
+    public void addFruits(Fruits fruits) {
+
+        listOfFruits.add(fruits);
+    }
+
+    public List<Fruits> getFreshFruits(LocalDate date) {
+        List<Fruits> listOfFresh = new ArrayList<>();
+        for (Fruits fruit : listOfFruits) {
+            if (fruit.shelfLife(date)) {
+                listOfFresh.add(fruit);
+            }
+        }
+        listOfFresh.forEach(System.out::println);
```

Comment:
better to call method println in external class, and print List that you return

Reply:
Same to other